neqo-transport/src/connection.rs

```diff
@@ -1261,6 +1279,10 @@ impl Connection {
                 SentPacket::new(now, ack_eliciting, tokens, packet.len(), in_flight),
             );
 
+            if self.first_1_rtt_pn.is_none() && epoch == 3 {
+                self.first_1_rtt_pn = Some(hdr.pn);
+            }
```

Comment:
In the key update changes, packet numbers are already tracked by space.  Maybe you could use that information instead.

Reply:
I will drop this and use HANDSHAKE_DONE and keys discarding. 

I wanted to land this code before we switch version to 25 and turn it on on Nightly (this problem is a blocker because it make use keep connection alive forever). The version switch PR landed so that is late not.... I will update mozilla-central when version 25 is good enough for interoperability (HANDSHAKE_DONE is blocking one)